### DISCUSSION TITLE
The harmful nature of open and closed source tools

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#101] [Past Thinking Caps]
As a user of both open and closed source tools, I think it is very important to compare and contrast them in terms of the harm they can (and do) produce. I think that there is a lot of disconnect between users on both sides which occurs because there are people who only have access to open source platforms while others solely use closed source. As we've discussed in this class, harm to civil society can be perpetuated through mapping and data as utilized by corporations, governments, etc. These users typically produce maps or other outputs that service their end goal without taking into consideration other people or even the truth of the situation as a whole. This can be combated through the application of open sources to support participatory mapping and critical cartography to publicize community voices and narratives. While I think that availability/cost is a large factor in what makes closed source the more negative choice between the two, it is also important to take "over" availability into consideration concerning open source. Open source platforms are free, usually easy(ish) to use, and don't often change drastically over time but the fact that ANYONE can use them introduces problems in whether these platforms are used more often in positive or negative ways. An easy summary of my thoughts of the harmful nature of both is that closed sources are often used to exploit people/communties/cultures/etc whereas on the other side it is people who typically take advantage of or exploit open sources in order to put out negative messages. I think comparing the two types of platforms/applications goes to show that even the most intentionally positive and helpful platforms (open sources) are subject to users that don't share the same ideals.

**Comment by rachan2023** [comment]
I share the same sentiment about closed and open source tools. Closed source tools are useful because they are user friendly and have a lot of functionality, but they are often expensive. The cost can prevent people who are not financially stable from creating digital projects, such as maps,  or collaborating with one another, which is especially important when trying to promote social change and giving voice to under-served communities. Open source tools do combat these issues, but the fact that these tools are widely available for anyone to use makes these tools beneficial and dangerous. Personally, I use open source tools for academic projects because they are free and easy to use as opposed to closed source tools. In addition, open source tools create a collaborative environment and facilitates participatory mapping, which I am an advocate of. I am aware of the Tragedy of the Commons, so when I do use these tools, I use them with good intentions and I am conscious that these tools are provided to me for free because a collective of developers are actively contributing these tools. However, there are people can misuse these tools and create applications that may hurt certain communities or take advantage of these tools because they are free. Overall, open source tools were created with good intentions, but bad actors make these tools harmful.
